src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -210,11 +210,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                          "_embedVbCoreRuntime and _suppressEmbeddedDeclarations are mutually exclusive")
         End Sub
 
-        Private Sub New(other As VisualBasicCompilationOptions)
+        Friend Sub New(other As VisualBasicCompilationOptions)
             MyClass.New(
                 outputKind:=other.OutputKind,
                 reportSuppressedDiagnostics:=other.ReportSuppressedDiagnostics,
-                moduleName:=other.ModuleName,
+                ModuleName:=other.ModuleName,
```

Comment:
Parameter name on other constructor should be `moduleName` rather than `ModuleName`.